src/main/java/org/runejs/client/Landscape.java

```diff
@@ -96,9 +96,9 @@ public static void loadRegion() {
                         int offsetY = -Class26.baseY + (0xff & ISAAC.mapCoordinates[pointer]) * 64;
                         int offsetX = -MovedStatics.baseX + 64 * (ISAAC.mapCoordinates[pointer] >> 8);
                         byte[] is = RSString.terrainData[pointer];
-                        if(FileOperations.FileExists("./data/maps/" + LinkedList.anIntArray1071[pointer] + ".dat")) {
-                            System.out.println("reading file: " + "./data/maps/" + LinkedList.anIntArray1071[pointer] + ".dat");
-                            is = FileOperations.ReadFile("./data/maps/" + LinkedList.anIntArray1071[pointer] + ".dat");
+                        if(FileOperations.FileExists("./data/maps/" + LinkedList.terrainDataIds[pointer] + ".dat")) {
+                            System.out.println("reading file: " + "./data/maps/" + LinkedList.terrainDataIds[pointer] + ".dat");
```

Comment:
Do we need this log in?

Reply:
I don't want to remove it here (was here before)